### PR TITLE
fix: legend default config for multiple chart instances

### LINKED ISSUE #70: Label position wrong with multiple instances of builder
## _When_ does this occur?

When two builder instances are put on the same page and have differing legend positions the position of the last loaded instance will be used for all instances upon after refresh.

## _Where_ on the platform does it happen?

Both on LEX pages and on Community pages. 

## _How_ do we replicate the issue?

Create two instances of a builder on the same page. Select diferent legend positions. Save the page and refresh.

## Expected behavior (i.e. solution)

Each instance should have the assigned position for that instance..

### PR DESCRIPTION
**Pull request by victorgz**
## Description

Fixes #70 

## How Has This Been Tested?

Local tests + previsualization of sample app

## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: __tests__/unit/chartConfigService.test.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ describe('c-chart-config-service', () => {
   test(`Default configuration`, () => {
```
Comment (scolladon) [inline __tests__/unit/chartConfigService.test.js:5]: **suggestion:** maybe a test to ensure call to defaultConfiguration end with different object

The test could compare two defaultConfiguration call this way:
- === does not work
- == work

Comment (victorgz) [reply __tests__/unit/chartConfigService.test.js:5]: Yes good idea !

Comment (scolladon) [reply __tests__/unit/chartConfigService.test.js:23]: What I meant is that we should test that calling `ChartConfigService.defaultConfiguration()` does not return a shallow copy but a new Object instance (with the same value, right, but a new instance of the same data)

File: __tests__/unit/chartConfigService.test.js [hunk 2]
```diff
@@ -2,10 +2,17 @@ import ChartConfigService from 'c/chartConfigService';
 import { ATTRIBUTE_CARTESIAN_AXES } from 'c/constants';
 
 describe('c-chart-config-service', () => {
+  test(`Get new instance of default configuration`, () => {
+    const firstInstance = ChartConfigService.defaultConfiguration();
+    const secondInstance = ChartConfigService.defaultConfiguration();
+
+    expect(firstInstance).toStrictEqual(secondInstance); // Objects should have the same type and structure
+    expect(firstInstance === secondInstance).toBe(false); // Objects should not be a shallow copy but a new instance
```
Comment (scolladon) [inline __tests__/unit/chartConfigService.test.js:10]: **suggestion:** could be rewritten using `not.toBe`
```suggestion
    expect(firstInstance).not.toBe(secondInstance); // Objects should not be a shallow copy but a new instance
```

**Pull request merged**